pkg/kubelet/kubelet_network_linux.go

```diff
@@ -41,21 +41,6 @@ func (kl *Kubelet) initNetworkUtil() {
 // 2. 	In nat table, KUBE-MARK-MASQ rule to mark connections for SNAT
 // 	Marked connection will get SNAT on POSTROUTING Chain in nat table
 func (kl *Kubelet) syncNetworkUtil() {
-	if kl.iptablesMasqueradeBit < 0 || kl.iptablesMasqueradeBit > 31 {
-		klog.Errorf("invalid iptables-masquerade-bit %v not in [0, 31]", kl.iptablesMasqueradeBit)
-		return
-	}
-
-	if kl.iptablesDropBit < 0 || kl.iptablesDropBit > 31 {
-		klog.Errorf("invalid iptables-drop-bit %v not in [0, 31]", kl.iptablesDropBit)
-		return
-	}
-
-	if kl.iptablesDropBit == kl.iptablesMasqueradeBit {
```

Comment:
where is this `kl.iptablesDropBit == kl.iptablesMasqueradeBit` validated, 
I can see the 2 first checks here https://github.com/kubernetes/kubernetes/blob/fe8aa6eba8026f9562cfbd9d6ceee51a0939f4c4/pkg/kubelet/apis/config/validation/validation.go#L75-L80
Should we add this to validation too if we remove it here?

Reply:
scratch my first comment, I found it,  it's here https://github.com/kubernetes/kubernetes/blob/fe8aa6eba8026f9562cfbd9d6ceee51a0939f4c4/pkg/kubelet/kubelet.go#L368